CanvasPlusPlayground/Schema/Persistable/Course.swift

```diff
@@ -191,7 +191,8 @@ final class Course: Cacheable {
     // We cannot use `Color` directly because it needs to conform to `PersistentModel`
     var rgbColors: RGBColors?
     var isFavorite: Bool?
-
+    var nickname: String?
+    
```

Comment:
I'd just add a property here so you don't have to keep using the nil coalescing (`??`) operator:

```swift
var displayName: String {
    nickname ?? name ?? "Unknown Name"
}
```